application/src/main/data/json/edge/instructions/install/docker/instructions.md

```diff
@@ -4,9 +4,26 @@ Here is the list of commands, that can be used to quickly install ThingsBoard Ed
 
 Install <a href="https://docs.docker.com/engine/install/" target="_blank"> Docker CE</a> and <a href="https://docs.docker.com/compose/install/" target="_blank"> Docker Compose</a>.
 
-#### Running ThingsBoard Edge as docker service
+#### Step 1. Running ThingsBoard Edge
 
-Create docker compose file for ThingsBoard Edge service:
+Here you can find ThingsBoard Edge docker image:
+
+<a href="https://hub.docker.com/r/thingsboard/tb-edge" target="_blank"> thingsboard/tb-edge</a>
+
+#### Step 2. Choose Queue and/or Database Services
+
+ThingsBoard Edge is able to use different messaging systems/brokers for storing the messages and communication between ThingsBoard services. 
```

Comment:
Starting from 4.0 Kafka and in-memory only.
Please update accordingly.
Additionally, please add statement, that we are going to use SQL only approach and in-memory queue in this installation guide.
And add that for other details please refer to official docs site.